src/TodoInfo.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from './types/Todo';
+
+interface Props {
+  todo: Todo;
+  selectedTodo: Todo | null;
+  selectTodo: (todoId: Todo | null) => void
+}
+
+export const TodoInfo: React.FC<Props> = ({
+  todo,
+  selectTodo,
+  selectedTodo,
+}) => {
+  return (
+    <>
+      <tr key={todo.id} data-cy="todo" className="">
+        <td className="is-vcentered">{todo.id}</td>
+        <td className="is-vcentered">
+          {todo.completed && (
+            <span className="icon" data-cy="iconCompleted">
+              <i className="fas fa-check" />
+            </span>
+          )}
+        </td>
+        <td className="is-vcentered is-expanded">
+          <p
+            className={cn({
+              'has-text-link': todo.completed === true,
+              'has-text-danger': todo.completed === false,
```

Comment:
you should use shortcuts for booleans
```suggestion
              'has-text-link': todo.completed,
              'has-text-danger': !todo.completed,
```